packages/siop-oid4vp/lib/__tests__/AuthenticationRequest.request.spec.ts

```diff
@@ -98,6 +98,26 @@ describe('create Request Uri should', () => {
         response_type: 'id_token',
         request_object_signing_alg_values_supported: [SigningAlgo.EDDSA, SigningAlgo.ES256],
         redirect_uri: EXAMPLE_REDIRECT_URL,
+        claims: {
```

Comment:
Why do we need this all of a sudden? claims are only required when it really is OID4VP. Not for SIOPv2

Reply:
removed again and tests are passing. I remember tests failing at some point, but something else must have been fixed by now

Reply:
Actually test "succeed when requesting with a valid PD" fails when I do not provide the claims on line 553
> TypeError: Cannot read properties of undefined (reading 'vp_token')